BUILDING.md

```diff
@@ -1,244 +1,102 @@
+### Setup a private ECR for development/testing
+It is recommended that you have some form of protected container registry to use for testing. Currently twoliter does not allow the transfer of locally stored kits between a local copy of bottlerocket-core-kit and bottlerocket. 
+For testing purposes you can either utilize mutable tags to allow overriding of multiple versions of a core kit as you test, or you can use immutable tags and continuously bump the core kit version via the `Twoliter.toml`. 
 
-```shell
-cargo make -e BUILDSYS_JOBS=4
+### Setup Infra.toml
+An `Infra.toml` file needs to be created and should have a definition of your vendor (container registry) in order to publish the kits you build. To do so make sure that the `Infra.toml` has the below.
 ```
-
-(You can use variant and arch arguments together, too.)
-
-#### Package licenses
-
-Most packages will include license files extracted from upstream source archives.
-However, in some rare cases there are multiple licenses that could apply to a package.
-Bottlerocket's build system uses the `Licenses.toml` file in conjunction with the `licenses` directory to configure the licenses used for such special packages.
-Here is an example of a simple `Licenses.toml` configuration file:
-
-```toml
-[package]
-spdx-id = "SPDX-ID"
-licenses = [
-  { path = "the-license.txt" }
-]
+[aws]
+regions = ["<region>"]
+[vendor.<vendor-name>]
+registry = "####.dkr.ecr.us-west-2.amazonaws.com"
 ```
-
-In the previous example, it is expected that the file `the-license.txt` is present in `licenses`.
-You can retrieve the licenses from a remote endpoint, or the local filesystem if you specify the `license-url` field:
-
-```toml
-[package]
-spdx-id = "SPDX-ID AND SPDX-ID-2" # Package with multiple licenses
-licenses = [
-  # This file is copied from a file system, and will be saved as `path`
-  { license-url = "file:///path/to/spdx-id-license.txt", path = "spdx-id-license.txt" },
-  # This file is fetched from an https endpoint, and will be saved as `path`
-  { license-url = "https://localhost/spdx-id-license-v2.txt", path = "spdx-id-license-2.txt" }
-]
+After the kit has been built locally the kit can be published to the provided vendor in Infra.toml. This operation can also be handled by the makefile.
 ```
-
-#### NVIDIA variants
-
-If you want to build any of the NVIDIA variants, you can follow these steps to prepare a `Licenses.toml` file using the [License for customer use of NVIDIA software](https://www.nvidia.com/en-us/drivers/nvidia-license/):
-
-1. Create a `Licenses.toml` file in your Bottlerocket root directory, with the following content:
-
-```toml
-[nvidia]
-spdx-id = "LicensesRef-NVIDIA-Customer-Use"
-licenses = [
-  { path = "LICENSE", license-url = "https://www.nvidia.com/en-us/drivers/nvidia-license/" }
-]
+make publish VENDOR=<vendor-name>
```

Comment:
Correct me if I am wrong. You need to rotate your docker credential with the ECR access. So it would be good to mention that it is needed before we can publish.
```
aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin ####.dkr.ecr.us-west-2.amazonaws.com
```

Reply:
Yup. I added this in. Thanks!